static/src/javascripts/projects/commercial/modules/dfp/load-advert.js

```diff
@@ -1,7 +1,32 @@
 // @flow
 import { Advert } from 'commercial/modules/dfp/Advert';
-import prebid from 'commercial/modules/prebid/prebid';
+import prebid from 'commercial/modules/header-bidding/prebid/prebid';
 import { markTime } from 'lib/user-timing';
+import a9 from 'commercial/modules/header-bidding/a9/a9';
+
+const slotFlatMap = (advert, hbSlot) => {
```

Comment:
Is there a name for this that would mean more in the use case to which it is applied? `flatMap` has special meaning in a couple of contexts:

* Scala
* A general implementation of dependent operations within an effect

Neither of these really apply here! the signature is very different and in particular, this function doesn't take another function as an argument. It's a bit closer to `flatten` but even there, this is a specific piece of logic for adverts and header bidding.

If there's a name to describe the logic this actually does (including the `fiddle with top-above-nav hbslot(s)` stuff), I think this would be much clearer!

Reply:
slotFlatMap doesn't say anything about what this does.

Maybe a better name would be 'forcedPrebidSlotSize'?

Reply:
agree it was not the right name, have now renamed to `forcedSlotSize` as is been used be both prebid & a9